src/NHSD.BuyingCatalogue.Identity.UserDatabase/Tables/Organisations.sql

```diff
@@ -1,7 +1,8 @@
 ﻿CREATE TABLE [dbo].[Organisations]
 (
-	[Id] UNIQUEIDENTIFIER NOT NULL PRIMARY KEY, 
-    [Name] VARCHAR(255) NOT NULL, 
-    [OdsCode] VARCHAR(8) NULL, 
-    [LastUpdated] DATETIME2 NOT NULL
-)
+	Id UNIQUEIDENTIFIER NOT NULL,
+    [Name] NVARCHAR(255) NOT NULL,
+    OdsCode NVARCHAR(8) NULL,
+    LastUpdated DATETIME2 NOT NULL DEFAULT GETUTCDATE(),
```

Comment:
I believe the last updated column should be a DATETIME2 (7)?

Reply:
The default precision is 7 so `datetime2` and `datetime(7)` are equivalent. However, it certainly wouldn't be a problem to use the latter for consistency – @RuthBerry96 can make the call on that one.

Reply:
Changed to DateTime2(7) to remain constant with the buyingcatalogue